app/assets/javascripts/users.js

```diff
@@ -1,17 +1,63 @@
 $(function() {
-  'use strict';
+  "use strict";
+  var custom_context_id;
+  var tooltipText;
 
-  $('body').on('ajax:success', '.users_pagination', function (e, data) {
-    $('#users_list').html(data);
+  $("body").on("ajax:success", ".users_pagination", function(e, data) {
+    $("#users_list").html(data);
     $('[data-toggle="tooltip"]').tooltip();
   });
 
-  $('#userStatusForm').on('ajax:success', function (e, data) {
-    $('#userMemberStatus').html(data);
+  $("#userStatusForm").on("ajax:success", function(e, data) {
+    $("#userMemberStatus").html(data);
     $('[data-toggle="tooltip"]').tooltip();
   });
 
-  $('#editUserStatusSubmit').click(function() {
-    $('#editStatusModal').modal('hide');
+  $("#editUserStatusSubmit").click(function() {
+    $("#editStatusModal").modal("hide");
+  });
+
+  $(".custom-context").bind("contextmenu", e => {
```

Comment:
Please note that jQuery `.bind` has been deprecated as of jQuery 3.0 (we're at 3.4.1).  We should use `.on1 instead.